docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -0,0 +1,137 @@
+---
+title: System.Net Metrics
+description: Review the metrics available for System.Net
+ms.topic: reference
+ms.date: 9/21/2023
+---
+
+# System.Net Metrics
+
+This article describes the networking metrics built-in for <xref:System.Net> produced using the
+<xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the alternate [EventCounters](event-counters.md) API,
+see [here](available-counters.md).
+
+- [Meter: `System.Net.NameResolution`](#meter-systemnetnameresolution) - Metrics for DNS lookups
+  * [Instrument: `dns.lookups.duration`](#instrument-dnslookupsduration)
+- [Meter: `System.Net.Http`](#meter-systemnethttp) - Metrics for outbound networking requests using HttpClient
+  * [Instrument: `http.client.open_connections`](#instrument-httpclientopen_connections)
+  * [Instrument: `http.client.connection.duration`](#instrument-httpclientconnectionduration)
+  * [Instrument: `http.client.request.duration`](#instrument-httpclientrequestduration)
+  * [Instrument: `http.client.request.time_in_queue`](#instrument-httpclientrequesttime_in_queue)
+  * [Instrument: `http.client.active_requests`](#instrument-httpclientactive_requests)
+
+## Meter: `System.Net.NameResolution`
+
+### Instrument: `dns.lookups.duration`
+
+| Name     | Instrument Type | Unit | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dns.lookups.duration` | Histogram | `s` | Measures the time taken to perform a DNS lookup. |
+
+| Attribute  | Type | Description  | Examples  | Presence |
+|---|---|---|---|---|
+| `dns.question.name` | string | The name being queried. | `www.example.com`; `dot.net` | Always |
+
+This metric measures the time take to make DNS requests. These requests can occur by calling
+<xref:System.Net.Dns.GetHostName> or indirectly wihtin higher level APIs on types such as <xref:System.Net.Http.HttpClient>.
+
+Available starting in: .NET 8
```

Comment:
Ideas for integrating a link to the definition in semantic conventions into this UI: There could be a bullet point list of details after the description. e.g.

* Available starting in: .NET 8
* Full specification: `dns.lookups.duration`

Reply:
Sounds fine to me. We won't be able to do this until the OTel docs are merged.

Reply:
I'm not planning to hold this PR until the OTel stuff is in, but we can always update it later.